torch/csrc/distributed/c10d/CUDASymmetricMemory-inl.h

```diff
@@ -322,8 +325,8 @@ __device__ __inline__ T add_bf16x2(T a, T b) {
   return T{};
 #else
   auto res = __hadd2(
-      *reinterpret_cast<__nv_bfloat162*>(&a),
-      *reinterpret_cast<__nv_bfloat162*>(&b));
+      *reinterpret_cast<__hip_bfloat162*>(&a),
```

Comment:
Better handle this in hipify.
Also this change is in CUDA block so I believe it will break CUDA build.

Reply:
oh woops, I accidentally commited the hipified file. i'll roll this back